enterprise/server/util/cpuset/cpuset_test.go

```diff
@@ -199,11 +199,13 @@ func TestMaxNumberOfLeases(t *testing.T) {
 	flags.Set(t, "executor.cpu_leaser.overhead", 0)
 	flags.Set(t, "executor.cpu_leaser.min_overhead", 0)
 	flags.Set(t, "executor.cpu_leaser.cpuset", "0")
+	// Don't log warnings to avoid spamming test logs.
+	flags.Set(t, "executor.cpu_leaser.warn_about_leaks", false)
 
 	cs, err := cpuset.NewLeaser()
 	require.NoError(t, err)
 
-	numLeases := cpuset.MaxNumLeases * 3
+	numLeases := cpuset.MaxNumLeases
```

Comment:
I think this was maybe set to max * 3 to ensure that even if you acquire more than the max, the number of leases is the max?